core/controllers/acl_decorators.py

```
@@ -2883,6 +2884,63 @@ def test_login(
+                attempt_id
+            )
+        except (
+            certificate_assessment_services.CertificateAssessmentAttemptNotFoundException
+        ) as e:
+            raise self.NotFoundException(str(e)) from e
+
+        if attempt.learner_id != self.user_id:
+            raise self.UnauthorizedUserException(
+                'You do not have permission to access this certificate '
+                'assessment attempt result.'
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)